code/modules/clothing/modular_armor/xenonauten.dm

```diff
@@ -178,6 +178,47 @@
 	attachments_allowed = list()
 
 
+	allowed = list(
+		/obj/item/weapon/gun,
+		/obj/item/tank/emergency_oxygen,
+		/obj/item/flashlight,
+		/obj/item/ammo_magazine,
+		/obj/item/storage/fancy/cigarettes,
+		/obj/item/tool/lighter,
+		/obj/item/weapon/baton,
+		/obj/item/restraints/handcuffs,
+		/obj/item/explosive/grenade,
+		/obj/item/binoculars,
+		/obj/item/weapon/combat_knife,
+		/obj/item/attachable/bayonetknife,
+		/obj/item/storage/belt/sparepouch,
+		/obj/item/storage/holster/blade,
+		/obj/item/storage/holster/belt,
+		/obj/item/weapon/energy/sword,
+	)
+
+/obj/item/clothing/suit/storage/modular/xenonauten/ballistic
```

Comment:
what is this typepath?
it should be like /obj/item/clothing/suit/modular/yourthinghere, if you want it to be like a modular armour type, or just /obj/item/clothing/suit/storage/yourthinghere if its going to stay as a suit/storage type suit